### ISSUE TITLE
how to fixed to AP  AND get csi message

### ISSUE THREAD

**Issue by gechangwei** [xieyaxiongfly/AtherosCSI_openWRT_ath9k#1]
hi: 
   i am very interesting your tool for openwrt, but i want to konw how to put it to the driver of ath9k.
   thanks.

**Comment by xieyaxiongfly** [comment]
You just replace this ath9k driver with the one in your openWRT code, then its done.

**Comment by gechangwei** [comment]
hi 亚雄：
   我已经把你openwrt版本csi的文件直接放到ath9k文件中，也已经编译成功，但是我想知道我怎么去获得CSI的信息，是需要在路由器上运行之前那个用户态的recv程序吗？还是需要再把用户态的程序也交叉编译到固件中来，谢谢？

**Comment by xieyaxiongfly** [comment]
Yes, you need to compile the user space applications and run it on your
openWRT system. You can use cross-compile to achieve that.

**Comment by wynter-wang** [comment]
@gechangwei Hello gechangwei! Which atheros nic did you use? I ran it on ar9344 successfully, and I could get csi data normally. But I had faced some strenge question, we can study it together! My email: wangtdada@163.com
